src/arglist.c

```diff
@@ -108,22 +108,22 @@ alist_expand(int *fnum_list, int fnum_len)
     // Don't use 'suffixes' here.  This should work like the shell did the
     // expansion.  Also, the vimrc file isn't read yet, thus the user
     // can't set the options.
```

Comment:
It seems this comment also needs to be moved.

Reply:
Yes. Updated the PR to move the comment.